libs/examples/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tslint.json",
+  "rules": {
+    "directive-selector": [true, "attribute", "dtExample", "camelCase"],
```

Comment:
you are using the camelCase attribute ;)
```suggestion
    "directive-selector": [true, "attribute", "dt-example", "camelCase"],
```

Reply:
camelCase is correct. Why would you wanna change it?

Reply:
you have to write the lint rule as followed ;)

Reply:
sorry you are right :)